server/app/views/LoginForm.java

```diff
@@ -129,7 +134,7 @@ private ContainerTag mainContent(Messages messages) {
     }
     content.with(alternativeLoginButtons);
 
-    String somethingElse = messages.at(MessageKey.CONTENT_ADMIN_LOGIN_PROMPT.getKeyName());
```

Comment:
that was... something else =)
thanks for renaming

Reply:
I think it's because the message is "Looking for something else?", haha